public/pages/DetectorResults/components/DetectorState/DetectorInitializing.tsx

```diff
@@ -34,14 +48,24 @@ export const DetectorInitializing = (props: DetectorInitializingProps) => {
       }
       body={
         <Fragment>
-          <p>
-            Based on your latest update to the detector configuration, the
-            detector is collecting sufficient data to generate accurate
-            real-time anomalies.
-          </p>
-          <p>
-            The longer the detector interval is, the more time this will take.
-          </p>
+          {!isInitOvertime
+            ? [
+                <p>
+                  Based on your latest update to the detector configuration, the
+                  detector is collecting sufficient data to generate accurate
+                  real-time anomalies.
```

Comment:
This does not seem true.  The detector may not receive any data in its shingle before isInitOverTime become true..

Reply:
This message is shown when detector is just created/started and being initialized. 

I think this message's point is: when initialization is started, and before `isInitOverTime` becomes true, _no matter data is collected or not_, we show it to user to let user know detector is collecting data.

If no data is collected, and it is overtime, then we show message below to let user know the reason and action item; if it is completed, we just show AnomalyResult page instead.

Reply:
Understood.  I feel "sufficient" is not correct as we are not certain.

Reply:
Yeah. You are right. I will remove `sufficient` to avoid confusion.